lib/jira/client/index.js

```diff
@@ -144,6 +148,36 @@ module.exports = async (jiraHost, gitHubInstallationId, logger) => {
   return client
 }
 
+/**
+ * Splits commits in data payload into chunks of 400 and makes separate requests
+ * to avoid Jira API limit
+ */
+const batchedCommitRequest = async (data, options, instance, installationId) => {
+  const filteredCommits = dedupCommits(data)
+
+  const commitChunks = []
+  while (filteredCommits.length) {
+    commitChunks.push(filteredCommits.splice(0, 400))
+  }
+
+  const batchedUpdates = []
+  commitChunks.forEach((commitChunk) => {
+    const repositoryObj = {
+      ...data,
+      commits: commitChunk
+    }
+
+    batchedUpdates.push(instance.post('/rest/devinfo/0.10/bulk', {
```

Comment:
the instance.post is near identical to the one we see above. The only difference is `respositoryObj` instead of `data`. I would be concerned these could drift. 

One question is ... when would we not have `data.commits`? Can we otherwise make these the same path somehow?